contracts/ThalesRoyale/ThalesRoyale.sol

```diff
@@ -316,50 +369,71 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
     /* ========== INTERNALS ========== */
 
     function _signUpPlayer(address _player, uint _position) internal {
```

Comment:
can you add possibility of `uint [] _positions` as a input param because i know in next season user can choose all positions as array. (also signed up with sUSD and RoyalePass NFT)

Example six rounds [1,2,1,1,2,1,2]